src/test/java/io/cdap/plugin/gcp/bigquery/action/BigQueryExecuteTest.java

```diff
@@ -0,0 +1,108 @@
+package io.cdap.plugin.gcp.bigquery.action;
```

Comment:
Add Copyright text and comment on top of class.

Reply:
Updated !